python-backend/app/api/reporting/resources/metabase.py

```diff
@@ -23,3 +23,32 @@ def get(self):
 
         dashboard_url = current_app.config['METABASE_SITE_URL'] + "/embed/dashboard/" + token.decode("utf8")
         return { 'dashboard_url': dashboard_url }
+
+class LandingPageGraphOneResource(Resource, UserMixin):
+    @requires_role_view_all
+    def get(self):
+        payload = {
+          "resource": {"dashboard": 164},
+          "params": {
+
+          }
```

Comment:
This can just be `"params": {}`. Could you update line 18 as well, please and thank you 😄